frontend/src/components/Dropdown.tsx

```diff
@@ -389,12 +447,12 @@ export function ControlledPopup({
             rounded 
             text-emphasis 
             shadow-lg`}
-          style={{ transform: 'translateY(calc(-100% - 5px))' }}
+          style={{ transform: "translateY(calc(-100% - 5px))" }}
```

Comment:
create a custom css for this, don't use style

Reply:
Is this necessary? I think custom CSS for a single element is redundant.

Reply:
Yes it is, we should follow a practice not to use style and important keyword if it is not needed. Should only use class for easier to debug if any error rise.

Reply:
Done, review global css.